### PR TITLE
Fixing 21965 # Reporting for Deployment Tests.

### LINKED ISSUE #21965: Deployment devtest failures not reported on Hudson
In the deployment_all, deployment_cluster_all tests , at least some failures are not reported in the junit report file.

## Environment Details
* GlassFish Version (and build number): Internal DevTest Issue
* JDK version: 1.8
* OS: All
* Database:

----------

## Problem Description
In the following tests - deployment_all, deployment_cluster_all, at least some failures are not reported in the junit report file
Example: http://gf-hudson.us.oracle.com/hudson/job/gf-generic-test-job/17488/ - Link seems to be not working. Failures in InheritanceAnnotationTest were not being reported. There might be more such cases.

## Steps to reproduce
Just run the deployment_all test and see the Report

## Impact of Issue
No Service Impact. Only devtest failure is not Reported.

### PR DESCRIPTION
**Pull request by naman223**
Fix #21965 

In the following tests - deployment_all, deployment_cluster_all, at least some failures are not reported in the junit report file.

### CODE REVIEW

**Comment by naman223** [comment]
@glassfishrobot Run CI tests please

File: appserver/tests/appserv-tests/devtests/deployment/run_test.sh [hunk 1]
```diff
@@ -169,5 +169,7 @@ case $OPT in
     list_test_ids;;
   run_test_id )
     run_test_id $TEST_ID ;;
+  deployment_all )
```
Comment (arindam-bandyopadhyay) [inline appserver/tests/appserv-tests/devtests/deployment/run_test.sh]: Why we need to explicitly define deployment_all target. Isn't it taken care by run_test_id

**Review by arindam-bandyopadhyay — CHANGES_REQUESTED** [review]
I don't think we need to explicitly add deployment_all target in the deployment run_test.sh

File: appserver/tests/appserv-tests/devtests/deployment/run_test.sh (hunk 1 above)
Comment (naman223) [inline appserver/tests/appserv-tests/devtests/deployment/run_test.sh]: I added for easy to run the tests locally. 
No, run_test_id is doing some intialization which I don't want locally so just added target, I set S1AS & APS, it works fine so added.

**Review by arindam-bandyopadhyay — APPROVED** [review]

**Pull request merged**